content/docs/pipelines/sdk/client_usage.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "Use Kubeflow Pipelines API Client"
+description = "KFP SDK version >= 0.2.5 is required"
+weight = 30
++++
+
+This page includes a quick example on how to create a pipeline and a pipeline version using KFP API client.
+
+
+## Before you start
+
+* Install the [Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/). The following requires KFP SDK version equal to or greater than 0.2.5. You can test your installed version using
+```
+pip list | grep kfp
+```
+
+## Example 1: create a pipeline and then create a new version under it
```

Comment:
I would recommend splitting this into a heading for examples, and a heading for this example. For example:

## Examples

Use the following examples to learn more about the Kubeflow Pipelines SDK.

### Using the SDK to create a pipeline and a pipeline version

Reply:
I am now thinking that maybe I should just add one example here since I found other examples in the tutorial directory. So if I have other examples I probably will have a dedicated file to each of them.

Reply:
done